Marlin/src/HAL/TEENSY40_41/timers.h

```diff
@@ -36,13 +36,13 @@
 typedef uint32_t hal_timer_t;
 #define HAL_TIMER_TYPE_MAX 0xFFFFFFFE
 
-#define GPT_TIMER_RATE F_BUS_ACTUAL   // 150MHz
+#define GPT_TIMER_RATE F_BUS // 150MHz (Can't use F_BUS_ACTUAL because it's extern volatile)
```

Comment:
Note that this is only 1/4 of the full `F_CPU` speed, but it's unclear whether we can run interrupt timers at such a high resolution, or need to. All counts would require two extra bits.

Since `F_BUS_ACTUAL` is `volatile`, perhaps the system clock speed is able to change dynamically, or maybe this is only done under program control. If Marlin is able to keep the clock at a constant rate then we should be ok just using `F_BUS`, which is provided by the Teensy platform. I'm not sure yet how `F_CPU` and `F_BUS` are defined, since you can change the CPU speed with PlatformIO option `board_build.f_cpu`. If Teensy isn't picking up `board_build.f_cpu` then we may add our own `-DF_CPU` or some other define that Teensy platform is expecting.